aiosmtpd/tests/test_smtp.py

```diff
@@ -175,6 +175,17 @@ async def handle_exception(self, error):
         raise ValueError('ErroringErrorHandler test')
 
 
+class ErroringHandlerConnectionLost:
+    error = None
+
+    async def handle_DATA(self, server, session, envelope):
+        raise ConnectionResetError('ErroringHandlerConnectionLost test')
+
+    async def handle_exception(self, error):
+        self.error = error
+        raise ConnectionResetError('ErroringHandlerConnectionLost test')
```

Comment:
Is raise necessary? I mean, at this point, you're just trying to ensure that `ErroringHandlerConnectionLost` is not called, why the `raise`?

Reply:
It's really not in use here. I've removed it.